cagallo/book-club-api
# Fix auto increment bug

## What Features are Being Added?

- None

## What is Being Refactored?

- Refactor table seeding to restart auto increment back to 1 when `knex seed:run` happens

## Where should the reviewer start?

- seeds/books_favorites.js

## How should this be manually tested?

- run `knex seed:run` 
- check favorites table

## What are the relevant tickets?

- Fix auto increment bug when seeding data tables

## Any Other Notes/Questions/Problems?

- None at the moment!
